Story/RainMeadow.StoryHooks.cs

```diff
@@ -3,6 +3,7 @@
 using HUD;
 using System.Linq;
 using System;
+using static Sony.PS4.SaveData.Dialogs;
```

Comment:
?

Reply:
dude I have no idea lol stupid things get added automatically.

Reply:
happened b/c of visual studio intellesense and auto fills. Just look at the PR diff after you put it up and make the corrections.